src/Product/Receipt/ReceiptV5LineItem.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Mindee\Product\Receipt;
+
+use Mindee\Parsing\Common\SummaryHelper;
+use Mindee\Parsing\Standard\FieldConfidenceMixin;
+use Mindee\Parsing\Standard\FieldPositionMixin;
+
+/**
+ * List of line item details.
+ */
+class ReceiptV5LineItem
```

Comment:
should explicitly implement the `Stringable` interface

> As of PHP 8.0.0, any class that contains a [__toString()](https://www.php.net/manual/en/language.oop5.magic.php#object.tostring) method will also implicitly implement the [Stringable](https://www.php.net/manual/en/class.stringable.php) interface, and will thus pass type checks for that interface. Explicitly implementing the interface anyway is recommended.

https://www.php.net/manual/en/language.oop5.magic.php#object.tostring

```suggestion
class ReceiptV5LineItem implements Stringable
```

Reply:
Doesn't exist in 7.4 :upside_down_face: